canonical/charmhub.io
# Fix charmpage breaking when no architecture and bases

## Done
- Fix charm page breaking when no architecture and bases
## How to QA
- Go to https://charmhub-io-1474.demos.haus/landscape-scalable and ensure it displays properly without any error
## Issue / Card
Fixes #

## Screenshots
